drivers/remote/driver.go

```diff
@@ -72,5 +55,5 @@ func (d *driver) Leave(nid, eid types.UUID, options map[string]interface{}) erro
 }
 
 func (d *driver) Type() string {
-	return d.networkType
+	return remoteNetworkType
```

Comment:
Similar to the comment above. What is the proposed alternative to this functionality of returning a proper `networkType` for a remote driver ?